tests/01-ct.sh

```diff
@@ -40,13 +40,13 @@ function get_container_metadata {
   log "CLIENT_IP: $CLIENT_IP"
   CLIENT_IP4=$(docker inspect --format '{{ .NetworkSettings.Networks.cilium.IPAddress }}' client)
   log "CLIENT_IP4: $CLIENT_IP4"
-  CLIENT_ID=$(cilium endpoint list | grep id.client | awk '{ print $1}')
+  CLIENT_ID=$(cilium endpoint list -o jsonpath="${CILIUM_JSONPATH_FMT}" | grep id.client | awk '{ print $1}')
```

Comment:
If JSON / JSONPath output is going to be used, then a tool like `jq` should be used for parsing the output instead of `grep` and `awk`. This applies to most of these commands here.

Reply:
I agree, I didn't think we had jq installed and I tried to stick to the style used before (parse columns with awk). If you say we have jq, I shall change to using it.